kernel/aster-nix/src/vm/vmo/mod.rs

```diff
@@ -610,24 +436,25 @@ impl Vmo_ {
         Ok(())
     }
 
-    /// Determine whether a page is committed.
+    /// Determines whether a page is committed.
     pub fn is_page_committed(&self, page_idx: usize) -> bool {
-        self.pages.with(|pages, size| {
-            pages
-                .load((page_idx + self.page_idx_offset) as u64)
-                .is_some()
-        })
+        self.pages
+            .with(|pages, size| pages.load(page_idx as u64).is_some())
```

Comment:
If the variable `size` is never used, you can use an underscore instead, like `with(|pages, _|`. You can also check other code.